internal/commands/secrets/flags.go

```diff
@@ -0,0 +1,11 @@
+package secrets
```

Comment:
this file can probably go away ... just put these `const` declarations into the `input` file for this command

Reply:
I was thinking it could be reused for `secrets update` and `secrets delete`. Though you bring up a good point that maybe we could have a generic inputs file for all of those instead of create_inputs, update_inputs, etc?

Reply:
Possibly, but for `secrets delete` I'm planning on having either the `ID` or `Name` as a flag, which might not be reusable with other `delete_inputs`

Reply:
Ah ok I see that now. IMO the best route would to follow what Nick said and each command will have it's own inputs, even if update and delete could reuse that code. Keeps things separate so I like that trade off

Reply:
yeah, with the nature of some of these commands, we are just inevitably going to see a lot of repetition.  but we should embrace that!  especially because things like flags and what not aren't especially tricky concepts, we can afford that duplication to keep things decoupled